File: Programmers/Lv0/A강조하기.cpp
#include <string>
#include <vector>
using namespace std;

string solution(string myString) {
    for (char &ch : myString)
        if (ch == 'a')
            ch = toupper(ch);
        else if (isupper(ch) && ch != 'A')
            ch = tolower(ch);
    return myString;
}

File: Programmers/Lv0/A강조하기.py
def solution(myString):
    answer = ""
    for ch in myString:
        if 'B' <= ch <= 'Z':
            ch = chr(ord(ch) + (ord('a') - ord('A')))
        elif (ch == 'a'):
            ch = chr(ord(ch) - (ord('a') - ord('A')))
        answer += ch
    return answer